doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -242,6 +244,118 @@ Learn how to write filters in this [filter pattern syntax doc](https://docs.aws.
 [KubDoc]: https://kubernetes.io/docs/concepts/cluster-administration/logging/
 [ConfigLink]: https://raw.githubusercontent.com/ray-project/ray/releases/2.4.0/doc/source/cluster/kubernetes/configs/ray-cluster.log.yaml
 
+### Method 2: Grafana Loki
+
+#### Deploy Loki monolithic mode:
+
+Loki’s Helm chart supports three deployment methods to fit different scalability and performance needs: Monolithic, Simple Scalable, and Microservices. Here, we will demonstrate the monolithic method. For details on each deployment mode, see the [Loki deployment](https://grafana.com/docs/loki/latest/get-started/deployment-modes/) modes documentation.
+
+Deploy the Loki deployment with the [Helm chart repository](https://github.com/grafana/loki/tree/main/production/helm/loki).
+
+```shell
+helm repo add grafana https://grafana.github.io/helm-charts
```

Comment:
This section installs Loki and Grafana separately. Is it possible to have any version compatibility issues between Loki and Grafana?

Reply:
Grafana v11.3 supports Loki v2.8+. 
https://grafana.com/docs/grafana/latest/datasources/loki/